src/ImageSharp/Formats/Jpeg/JpegMetadata.cs

```diff
@@ -1,13 +1,36 @@
 // Copyright (c) Six Labors.
 // Licensed under the Apache License, Version 2.0.
 
+using System;
+using SixLabors.ImageSharp.Formats.Jpeg.Components;
+
 namespace SixLabors.ImageSharp.Formats.Jpeg
 {
     /// <summary>
     /// Provides Jpeg specific metadata information for the image.
     /// </summary>
     public class JpegMetadata : IDeepCloneable
     {
+        /// <summary>
+        /// Luminance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? lumaQuantizationTable;
+
+        /// <summary>
+        /// Luminance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? chromaQuantizationTable;
+
+        internal double LumaQuality;
```

Comment:
According to JPEGsnoop implementation, we have 2 quality multipliers - luma & chroma. Two questions:
1. should we expose both as public properties?
2. do we even need quality for separate channels so we can encode using different multipliers?

Reply:
I think we should expose both.

The original Quality parameter could be read as an average of both when we have tables from the decoder and written to both when encoding against the standard tables. We can use the specific properties to apply more granular quality. 

I think we could also obsolete Quality

Reply:
Does current stylecop setup forces to obsolete usage to be an compilation error? Marking anything with `Obsolete(msg, error: false)` breaks local compilation on release setup.

Reply:
Yeah must do. We'll have to ensure that we refactor tests to use the obsolete method as little as possible and suppress the warning in the source where we need to test it.